pyproject.toml

```diff
@@ -1,7 +1,46 @@
 [build-system]
+  requires = [
+    "setuptools",
+    "setuptools_scm",
+    "wheel"
+  ]
+  build-backend = 'setuptools.build_meta'
 
-requires = ["setuptools",
-            "setuptools_scm",
-            "wheel"]
 
-build-backend = 'setuptools.build_meta'
+[project]
+  name = "astropy_iers_data"
+  dynamic = ["version"]
+  description = "IERS Earth Rotation and Leap Second tables for the astropy core package"
+  readme = "README.rst"
+  requires-python = ">=3.8"
+  license = {file = "LICENSE.rst"}
+  keywords = ["astropy", "data", "IERS"]
+  authors = [
+    {name = "Astropy Developers", email = "astropy@mail.com"}
+  ]
+  classifiers = [
+    "Intended Audience :: Science/Research",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: BSD License",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3",
+  ]
+
+[project.urls]
+  homepage = "https://github.com/astropy/astropy-iers-data"
+  repository = "https://github.com/astropy/astropy-iers-data"
+  documentation = "https://github.com/astropy/astropy-iers-data"
```

Comment:
No need to put 3 times the same URL ?

Reply:
Debateably. There's no standard, but some common ones (which I've put here). We can refine the links somewhat.

e.g.

```python
  repsository = "https://github.com/astropy/astropy-iers-data/src/astropy_iers_data"
  documentation = "https://github.com/astropy/astropy-iers-data/README.rst"
```

Reply:
I'll update documentation to point to the README and remove homepage as there isn't really one as such (that is not the repository)